radis/asgi.py

```
@@ -12,9 +12,15 @@
 
 import os
 
-from django.core.asgi import get_asgi_application
-
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "radis.settings.development")
+
+# Initialize OpenTelemetry before Django loads to ensure all requests are traced
+from adit_radis_shared.telemetry import setup_opentelemetry  # noqa: E402
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)